gracc-collector.spec

```diff
@@ -7,6 +7,7 @@ License:        MIT
 URL:            https://gracc.opensciencegrid.org
 Source:         %{name}.tar.gz
```

Comment:
Yeah, I think this looks about right -- however, can we use something more descriptive than `_test`?  How about `_gracc_ci_test`?